shared/set_configs/EMN.json

```diff
@@ -14,11 +15,87 @@
     ]
   },
   "SET_CORRECTIONS": [{
-    "match": {
-      "multiverseid": 414497
+      "match": {
+        "imageName": "brisela, voice of nightmares1"
+      },
+      "replace": {
+        "number": "15b",
+        "names": [
+          "Bruna, the Fading Light",
+          "Brisela, Voice of Nightmares"
```

Comment:
I don't see any reason to make the front faces work differently than the back. By having all three names on each part, we have an easy and consistent way of referring to the other parts of the card, as well as determining which part the current card is.

Having two entries for the back would be fine with me as long as the names arrays are kept complete, since for those of us who don't care about physical cards they would just be duplicates which could be safely ignored.